Translations update from Weblate

ubports-weblate opened the pull request:
Translations update from [Weblate](https://translate.ubports.com/projects/ubports/indicator-session/) for UBports/indicator-session.

GermanAizek reviewed (commented):
ok
Most words match through the translator.

Lexa307 reviewed (commented):
Nice google translate

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/es/

Translated using Weblate (Portuguese)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/pt/

Translated using Weblate (Swedish)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/sv/

Translated using Weblate (Swedish)

Currently translated at 90.0% (27 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/sv/

Translated using Weblate (Arabic)

Currently translated at 46.6% (14 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/ar/

Translated using Weblate (Greek)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/el/

Translated using Weblate (Sardinian)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/sc/

Translated using Weblate (German)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/de/

Manual repair

Translated using Weblate (Thai)

Currently translated at 66.6% (20 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/th/

Translated using Weblate (Kashubian)

Currently translated at 56.6% (17 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/csb/

Translated using Weblate (Hebrew)

Currently translated at 100.0% (30 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/he/

Translated using Weblate (Amharic)

Currently translated at 96.6% (29 of 30 strings)

Translation: UBports/indicator-session
Translate-URL: https://translate.ubports.com/projects/ubports/indicator-session/am/
